### PR TITLE
	fix multiaccount methodes

### PR DESCRIPTION
**Pull request by Furtif**
## fix bug if accounts.db deleted and recreated

### CODE REVIEW

File: PoGo.NecroBot.Logic/MultiAccountManager.cs [hunk 1]
```diff
@@ -362,18 +362,29 @@ public void UpdateLocalAccount(Account current, bool save = true)
...
+                L = item.Username.Length;
+                if (L > maxL) { maxL = L; }
+            }
+
             foreach (var item in Accounts)
             {
-                var L = item.Nickname.Length;
                 var SP = "";
+                var user = !string.IsNullOrEmpty(item.Nickname) ? item.Nickname : item.Username;
+                var L = user.Length;
```
Comment (Lord-Haji) [inline PoGo.NecroBot.Logic/MultiAccountManager.cs]: Use q different name other than ``L``

Comment (Furtif) [inline PoGo.NecroBot.Logic/MultiAccountManager.cs]: ready

**Comment by TheWizard1328** [comment]
L is just a temp var for Length of Nickname or username

Wizard Worxx
Where the magic of software begins.
Robert Tauber
(780) 236-1542
www.Facebook.com/WizardWorxx13

**Comment by Furtif** [comment]
renamed var ready

**Pull request merged**